timesketch/lib/analyzers/yetiindicators.py

```diff
@@ -0,0 +1,129 @@
+"""Index analyzer plugin for Yeti indicators."""
+from __future__ import unicode_literals
+
+from flask import current_app
+import requests
+
+from timesketch.lib.analyzers import interface
+from timesketch.lib.analyzers import manager
+from timesketch.lib import emojis
+
+
+def build_query_for_indicators(indicators):
```

Comment:
This only get Yeti domain indicators. Do you plan on supporting more types? e.g. file hashes?
If that is the case we should probably have a mapping between Yeti indicator type and ES field to query. Then we can build this up dynamicaly in this function. WDYT?